Document `graphql.extensions.modifyUknownField` #791 (#792)
enonic/app-guillotine: docs/extending/creation-callbacks.adoc

}
----

> NOTE: By default, Guillotine will throw an error if it encounters an unknown field during modification of fields of some GraphQL type. You can customize this behavior by setting the `graphql.extensions.modifyUnknownField` property in the <<../configuration#, configuration>>.

Reviewer: ```suggestion
NOTE: By default, Guillotine will throw an error if it tries to modify an unknown field. You can customise this behaviour by setting the `graphql.extensions.modifyUnknownField` property in the <<../configuration#, configuration>>.
```
